impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -781,7 +813,7 @@ define([
         }
       },
       {
-        priority: -2,
```

Comment:
i did it here with -2 in a way we still have the -1 to use in the plugins. What makes sense here?